src/common/utils.js

```diff
@@ -100,10 +103,32 @@ export function copyToClipboard(data) {
   textField.remove();
 };
 
+/* Returns date set to local timezone and in format of locale */
+export function formatLocalDate(date, isUTC, timezone) {
```

Comment:
Can we extract a common method to calculate local time (and handle the case for utc) and have two other functions to get the format - getStandardFormat, getLocaleFormat